config.yml

```diff
@@ -56,7 +55,7 @@ production:
     env: production
     forceSSL: true
     trustProxy: true
-    baseUrl: 'https://ec2-manager.herokuapp.com/v1'
```

Comment:
I was under the impression that the base url was supposed to have the `/v1` in it